_includes/head.html

```diff
@@ -3,8 +3,13 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   {% if page.layout == "eip" %}
-    <title>EIP-{{ page.eip }}: {{ page.title | escape }}</title>
-    <meta property="og:title" content="EIP-{{ page.eip }}: {{ page.title | escape }}" />
+    {% if page.category == "ERC" %}
+      <title>EIP-{{ page.eip }}: {{ page.title | escape }}</title>
+      <meta property="og:title" content="EIP-{{ page.eip }}: {{ page.title | escape }}" />
+    {% elsif page.category != "ERC" %}
```

Comment:
There's no `else`? O.o

Reply:
I don't know if there is or not, and I couldn't find any documentation saying there is or is not. If it aint broke, don't fix it.

Reply:
https://github.com/ethereum/EIPs/pull/6017/files/40207fe4b72a1cc01c6e580787e769cefd99b548#diff-a88dab7bceb531aea80805e037c9a03ba58430da047369e82ba8a7a3270a3af1R9 :expressionless: